app/Models/BeatmapLeader.php

```diff
@@ -26,6 +26,7 @@ public static function sync(int $beatmapId, int $rulesetId): void
     {
         $searchParams = ScoreSearchParams::fromArray([
             'beatmap_ids' => [$beatmapId],
+            'is_legacy' => false,
```

Comment:
Do solo scores from stable have a `build_id`?

Reply:
no, it's what's being checked for is_legacy in the indexer itself as well.

Reply:
nvm, this is a different leader from what I was thinking of 🤨